subseas_flex_fcst/maproom.py

```diff
@@ -45,27 +49,28 @@
 
 APP.layout = layout.app_layout
 
-
-def read_cptdataset(y_transform=False):
-
-    fcst_mu = cptio.open_cptdataset(Path(DATA_PATH, Path(CONFIG["forecast_mu_file"])))
+#Should I move this function into the predictions.py file where I put the other funcs?
+#if we do so maybe I should redo the func to be more flexible since it is hard coded to read each file separately..
+def read_cptdataset(leadTime, startDate, y_transform=CONFIG["y_transform"]):
```

Comment:
Possibly... or make it an issue. The key here is that is we are going to use the same app for other cpt forecasts (biweekly, seasonal...) so all the functions associated with reading will either have to be more fancy to accommodate different cases, or have counterparts for the different cases, and the config file will tell which case we're in.
I am just not sure what shape that will all take as I review this now. So it may make sense to leave to other PR.

Reply:
I think it is best to make a separate PR for now; I have made a new issue for this.